README.md

```diff
@@ -1 +1,60 @@
-# try-bootstrap
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Portfolio Webpage
+
+> This project is meant to be a Bootstrap exercise from the Microverse Program's Module 1.
+
+The project consists in a mobile and desktop skeleton for a portfolio webpage. Feel free to try it out and explore the code as much as you like!
+
```

Comment:
Hello @Kalunge,
You have been doing an incredible job on the project so far👏👏
The project is very well structured😃😃
 However the project would be better if you Updated the readme file with the project screenshot. What do you think about that?

Happy Coding!👏👏👏